src/type/schema.js

```diff
@@ -204,7 +207,7 @@ export class GraphQLSchema {
     if (!possibleTypeMap[abstractType.name]) {
       const possibleTypes = this.getPossibleTypes(abstractType);
       invariant(
-        Array.isArray(possibleTypes),
+        Array.isArray(possibleTypes) && possibleTypes.length > 0,
```

Comment:
This is a late checking invariant, not part of the scheme validation. This change shouldn't be necessary

Reply:
@leebyron this was only necessary for getting the schema tests working

Reply:
Fully agree. The invariant can go away because `getPossibleTypes` is now guaranteed to return an array.
The test can go away because there's no way `isPossibleType` can produce an error.